ci/scripts/autobump-dependencies.py

```diff
@@ -18,6 +18,13 @@
 from bs4 import BeautifulSoup
 from git import Repo
 
+# Pinned versions for the resolvers. See use at the bottom.
+KEEPALIVED_VERSION = "2.2"
+SOCAT_VERSION = "1.8"
```

Comment:
Do we need to pin to 1.8? Wouldn't 1 be sufficient?

Reply:
fair point.